cfme/tests/openstack/infrastructure/test_provider.py

```diff
@@ -12,18 +13,27 @@
 ]
 
 
+@pytest.fixture(scope='module')
+def provider_recreate():
+    yield
+    uc = get_crud('uc_ci')
```

Comment:
unacceptable for a test, we can't hardcode provider keys that are internal to RH.  Use the provider fixture so you have the provider object that was actually parametrized for the test.